src/agor/tools/agent_handoffs.py

```
@@ -344,3 +345,152 @@ def generate_meta_feedback(
     
     # Apply detick processing for clean codeblock rendering
     return detick_content(meta_content)
+
+
+@dataclass
+class HandoffRequest:
```

Pylint (3.3.7)

[refactor] Too many instance attributes (13/7)

(R0902)